Makefile

```
@@ -252,9 +254,13 @@ wlroots-build:
 sway-build:
 	make meson-ninja-build -e APP_FOLDER=sway -e APP_VERSION=$(SWAY_VERSION)
 	sudo cp -f sway/contrib/grimshot /usr/local/bin/
+## Libs
+libvarlink-build:
+	make meson-ninja-build -e APP_FOLDER=libvarlink -e APP_VERSION=$(LIBVARLINK_VERSION)
 
 ## Apps
 kanshi-build:
+	libvarlink-build
```

```suggestion
kanshi-build: libvarlink-build
```
Fixes: https://github.com/luispabon/sway-ubuntu/pull/33/checks